index.html

```diff
@@ -51,6 +59,164 @@ <h1 class="header-title">Hey there. <br>I am Alex.</h1>
 
     </section>
 
+    <!-- Recent Work section -->
+<section class="recent-work">
+    <div class="section-title title">
+        <div class="title-container">
+            <h1>My Recent<br>Works</h1>
+            <span class="section-indicator"></span>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: only the main title is required to be wrapped in a `<h1>` tag. Other titles can be wrapped with any of the `hx` tags except the `h1`. This is so for semantic and conventional reasons. it also promotes your website's search optimization and accessibility. But still, this `h1` is there so kindly fix it :slightly_smiling_face: